pylib/openrazer/_fake_driver/razerfireflyhyperflux.cfg

```diff
@@ -12,6 +12,5 @@ files = r,device_serial,XX0000000068
         w,matrix_effect_reactive
         w,matrix_effect_spectrum
         w,matrix_effect_static
-        w,matrix_effect_wave
```

Comment:
Was wondering what was touched here when I ran the script to generate all fake drivers...Guess these were out of sync in master?

Reply:
possible that my generation script has a bug

Reply:
fixed in 6608825dc1780ffa2ed1d973a441e498c6a4e5cc